modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/LifecycleAwareConfigurationInstance.java

```diff
@@ -72,6 +75,14 @@
  */
 public final class LifecycleAwareConfigurationInstance extends AbstractInterceptable implements ConfigurationInstance {
 
+  private static final String ASYNC_TEST_CONNECTIVITY_TIMEOUT_PROPERTY =
+      MuleSystemProperties.SYSTEM_PROPERTY_PREFIX + "async.test.connectivity.timeout";
+
+  private static final Integer DEFAULT_ASYNC_TEST_CONNECTIVITY_TIMEOUT = 10000;
```

Comment:
still too small. My previous comment said that this was not big enough for SaaS services. 30s would be a good default